initiatives/submit-a-request/index.html

```diff
@@ -0,0 +1,433 @@
+                           id="foia-request_organization-address"
+                           type="text"
+                           disabled>
+
+                </div>
+
+                <div class="form-l_col form-l_col-1-2">
+                    <label class="form-label-header"
+                           for="foia-request_organization-city">
+                        City
+                    </label>
+                    <input class="input__long"
+                           id="foia-request_organization-city"
+                           type="text"
+                           disabled>
+                </div>
+
+                <div class="form-l_col form-l_col-1-2">
+                    <div class="form-l_col form-l_col-1-2">
+                        <label class="form-label-header"
+                               for="foia-request_organization-state">
+                            State
+                        </label>
+                        <div class="custom-select">
+                        <select class="custom-select_select"
+                                data-placeholder="Select the FOIA request category"
+                                id="foia-request_organization-state"
+                                disabled>
+                            <option value="">Please select</option>
+                            <option value="State">States to be added</option>
+                        </select>
+                        <span class="custom-select_icon
+                                     cf-icon
+                                     cf-icon-down"></span>
+                    </div>
+                    </div>
+                    <div class="form-l_col form-l_col-1-2">
+                        <label class="form-label-header" for="foia-request_organization-zip">
+                            Zip code
+                        </label>
+                        <input class="input__long"
+                               id="foia-request_organization-zip"
+                               type="text"
+                               disabled>
+                    </div>
+                </div>
+            </fieldset>
+
+            <div class="form-actions form-actions__right-on-med ">
+                <input class="btn form-actions_item"
+                       type="submit"
+                       value="Generate FOIA PDF">
+                <a href="#" class="btn btn__link form-actions_item">
+                    Sample Letter
+                    <span class="cf-icon cf-icon-pdf"></span>
+                </a>
+            </div>
+
+        </form>
+    </section>
+    <!-- End Form Module. -->
+    <section class="block">
+        <h3>How to submit</h3>
+        <p>
+            To minimize delays associated with mail delivery
+            or other possible misrouting of your request,
+            we recommend submitting your request by email.
+            However, we can also accept by fax or mail.
```

Comment:
That I agree sounds better, but it isn't in the cleared copy. 

cc @jameshupp.